crates/runtime/Cargo.toml

```diff
@@ -20,3 +20,13 @@ crossbeam-channel = "0.5"
 slight-events-api = { path = "../events-api" }
 slight-http-api = { path = "../http-api/" }
 tracing = { version = "0.1", features = ["log"] }
+slight-common = { path = "../common" }
+dyn-clone = "1"
+
+slight-kv = { path = "../kv" }
```

Comment:
Having the runtime crate have a dependency on the capabilities seems backward. We should discuss if there is another way we should be structuring these crates. 

I don't think we need to address this in this PR, but rather, something we should discuss in September.

Reply:
I am thinking about making all capabilities generic to `T: Ctx` where `Ctx` is a context trait that defines the behaviors of a context. 

```rust
struct Kv<T: Ctx> {
  ...
}
```

Reply:
Ah it was harder than I thought. I had some ideas about that requires more refactoring to the current design. 

Particularly, I want `HostState` to be cloneable.